var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -84,10 +84,27 @@ def build(self, spec, prefix):
         self.mech_name += self._lib_suffix
         self._build_mods('mod', link_flag, include_flag, 'mod_core')
 
-        # Store flags
-        self._incflags  = "-incflags '{}'\n".format(include_flag)
-        self._loadflags = "-loadflags '{}'\n".format(link_flag)
-
+        # Create rebuild script
+        with open("build_neurodamus.sh", "w") as f:
+            f.write("""#!/bin/sh
+set -e
+if [ "$#" -eq 0 ]; then
+    echo "******* Neurodamus builder *******"
+    echo "Syntax:"
+    echo "$0 <mods_dir> [add_include_flags] [add_link_flags]"
+    echo
+    echo "NOTE: mods_dir is literally passed to nrnivmodl. If you only have the mechanism mods"
+    echo " and wish to build neurodamus you need to include the neurodamus-specific mods."
+    echo " Under \$NEURODAMUS_ROOT/share you'll find the whole set of original mod files,"
+    echo " as well as the neurodamus-specific mods alone. You may copy/link them into your directory."
+    exit 1
+fi
+
+# run with nrnivmodl in path
+'{}' -incflags '{} '"$2" -loadflags '{} '"$3" "$1"
+""".format(str(which('nrnivmodl')), include_flag, link_flag))
```

Comment:
Could we move this big string into a global constant? Reading unindented code like this is not really ideal IMO.

Reply:
good point